pipeline_reviews/pipeline.py

```diff
@@ -0,0 +1,41 @@
+"""Pipeline script to run all reviews extracting, transforming and loading"""
+
+from datetime import datetime
+
+from psycopg2 import Error
+
+from extract import get_db_connection, get_game_ids, get_all_reviews, GamesNotFound
+from transform import transform_reviews, remove_unnamed
+from sentiment import isolate_non_stop_words, get_sentiment_per_game, get_sentiment_values
+from load import get_game_ids_foreign_key_values, move_reviews_to_db
+
+if __name__ == "__main__":
```

Comment:
only comment is maybe spacing out extract, transform and loading parts so it's readable